packages/card/card.scss

```diff
@@ -45,6 +45,9 @@ $card-padding-normal: $component-spacing--xl;
         box-shadow: $drop-shadow--large;
         transform: translateY(-0.125rem);
     }
+    html:not([data-mousenavigation]) &--clickable:focus-within {
```

Comment:
Du trenger vel ha med touch saken også ? Er en mixin for det, `keyboard-navigation` som bør brukes. Hvis det er med intensjon at denne ikke skal ha noe med toutch saken, så bør du utvide med en ny mixin som bare setter på mousenavigation. Vi må vekk fra å definere disse magiske klassene rundt omkring.

Reply:
Var ikke klar over at den eksisterte, flott!